# GazeboState
#
# Rikky Duivenvoorden 2017-01-31 -- For use in AER1217 Winter 2017
#
# Data communicates the position in a three element array [x, y, z] and the 
# rotation in the form of a quaternion in a four element array [x, y, z, w].

float64[3] position

float64[4] quaternion
